source/fuzz/transformation_flatten_conditional_branch.h

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SOURCE_FUZZ_TRANSFORMATION_FLATTEN_CONDITIONAL_BRANCH_H
+#define SOURCE_FUZZ_TRANSFORMATION_FLATTEN_CONDITIONAL_BRANCH_H
+
+#include "source/fuzz/transformation.h"
+
+namespace spvtools {
+namespace fuzz {
+
+class TransformationFlattenConditionalBranch : public Transformation {
+ public:
+  explicit TransformationFlattenConditionalBranch(
+      const protobufs::TransformationFlattenConditionalBranch& message);
+
+  TransformationFlattenConditionalBranch(
+      uint32_t header_block_id, bool true_branch_first,
+      const std::vector<protobufs::SideEffectWrapperInfo>&
+          side_effect_wrappers_info = {});
```

Comment:
I'm not keen on default parameters unless there is a really strong case for them, so can you make this non-default, and pass in {} for the cases where no wrappers are required?

I'm guessing that in your case this merely makes the test code a tad easier to write, and I don't think that's a strong case.

Reply:
I have made it non-default and updated the tests